synapse/storage/databases/main/user_erasure_store.py

```diff
@@ -14,11 +14,12 @@
 
 from typing import Dict, Iterable
 
-from synapse.storage._base import SQLBaseStore
+from synapse.storage.database import LoggingTransaction
+from synapse.storage.databases.main import CacheInvalidationWorkerStore
 from synapse.util.caches.descriptors import cached, cachedList
 
 
-class UserErasureWorkerStore(SQLBaseStore):
+class UserErasureWorkerStore(CacheInvalidationWorkerStore):
```

Comment:
I'm not sure about this change, is this due to it calling methods from that?

Reply:
Yes, `_invalidate_cache_and_stream` is the method in question. Sorry, should have written that down.